casanovo/denovo/model.py

```diff
@@ -139,6 +139,8 @@ def __init__(
             dropout=dropout,
             residues=residues,
             max_charge=max_charge,
+            enzyme_size=9,
```

Comment:
Yes it's the number of components for training. I agree and I think I should just make this a config entry. Could also read out from the config file as well and just count the number of entries in the enzymes list to avoid having to add another parameter.